File: POO/PHP/UberPool.php
<?php
    require_once('Car.php');

    class UberPool extends Car{
        public $brand;
        public $model;

        public function __constructor($license,$driver,$brand,$model){
            parent::__construct($license,$driver,4);
            $this->$brand = $brand;
            $this->$model = $model;
        }
    }
?>

File: POO/Java/UberPool.java
package POO.Java;

public class UberPool extends Car {
    private String brand;
    private String model;

    public UberPool(String license,Account driver,String brand, String model) {
        super(license,driver,4);
        this.brand=brand;
        this.model = model;
    }
}
